apps/main/src/loan/components/PageLlamaMarkets/Page.tsx

```diff
@@ -26,16 +30,22 @@ const { Spacing, MaxWidth, ModalHeight } = SizesAndSpaces
  * Page for displaying the lending markets table.
  */
 export const PageLlamaMarkets = () => {
-  const { data, isLoading, isError } = useLlamaMarkets() // todo: show errors and loading state
+  const { signerAddress } = useWallet()
+  const { data, isError, isLoading } = useLlamaMarkets(signerAddress)
   const bannerHeight = useStore((state) => state.layout.height.globalAlert)
   const headerHeight = useHeaderHeight(bannerHeight)
   usePageOnMount() // required for connecting wallet
   return (
     <Box sx={{ marginBlockEnd: Spacing.xxl }}>
-      {isLoading ? (
+      {!data && (!isError || isLoading) ? (
```

Comment:
why is simply using isLoading not sufficient? i'd expect isLoading to be false if there's no data but an error regardless

Reply:
on initial render isLoading is still false. I'll add a comment